cmd/zt_traverser_managed_disk_test.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import "testing"
+
+func TestLargeManagedDiskSnapshot(t *testing.T) {
+	//a := assert.New(t)
+	//
+	//// Set up for this test.
+	//// In Azure Portal create a managed disk of size 7.9 TB
+	//// Select 'Create Snapshot'
+	//// Under Snapshot Export
+	//// Click on Generate URL and paste that URL below as largeMDSnapshot
+	// largeMDSnapshot := "https://md-XXXXX.blob.storage.azure.net/XXXX/XXXX?snapshot=XXXXXXXXsv=2018-03-28&sr=b&si=XXXXXXX&sig=XXXXXXXXXX"
+	//serviceClientWithSAS := scenarioHelper{}.getBlobServiceClientWithSASFromURL(a, largeMDSnapshot)
+	//
+	//// Create a page blob client with OAuth
+	//blobClient, err := blob.NewClientWithNoCredential(largeMDSnapshot, nil)
+	//a.NoError(err)
+	//
+	//blobProps, err := blobClient.GetProperties(context.TODO(), nil)
+	//a.NoError(err)
+	//a.Nil(blobProps.LastModified)
+	//
+	//blobTraverser := newBlobTraverser(largeMDSnapshot, serviceClientWithSAS, ctx, true, false, func(common.EntityType) {}, false, common.CpkOptions{}, false, false, false, common.EPreservePermissionsOption.None(), false)
+	//blobDummyProcessor := dummyProcessor{}
+	//err = blobTraverser.Traverse(noPreProccessor, blobDummyProcessor.process, nil)
+	//a.NoError(err)
+}
+
+func TestLargeManagedDisk(t *testing.T) {
+	//a := assert.New(t)
```

Comment:
I think I prefer these tests existing in the E2E test framework instead of as a unit test. Instead of commenting you should be able to write code to skip the test and enable it based on a flag.

Reply:
Why do we want to keep these tests in comments?